listenbrainz_spark/recommendations/recording/create_dataframes.py

```diff
@@ -103,9 +103,12 @@ def save_dataframe_metadata_to_hdfs(metadata: dict, df_metadata_path: str):
         raise
 
 
-def describe_listencount_transformer():
-    """ Returns a human-readable description of the algorithm used to transform listen counts"""
-    return "No transformation applied to listen counts"
+def describe_listencount_transformer(use_transformed_listencounts):
+    """ Returns a human-readable description of the algorithm used to transform listen counts """
+    if use_transformed_listencounts:
+        return "No transformation applied to listen counts"
```

Comment:
I guess this string should be updated.

Reply:
Uh no, this PR only adds the switch for using playcounts or transformed listencounts. For this PR for both transformed listencounts and playcounts are same so the string is same in both cases. The next PR adds the transform function.

This is because we only merged the code to allow use of transformed listencount last time and not the actual transfer function as there was no switch to turn it off at that time and we wanted to play with both.

Reply:
Yes, I saw that in the latter PR. Thanks for clarifying. Lets proceed!